### PR TITLE
Second-Milestone: Update Animls Table

### PR DESCRIPTION
**Pull request by saluumaa**
In this Milestone I:- 

1. Add a species column 
2. Alter animals data and add new data
3. Add transactions to alter the animals' table
4. Take screenshots for transaction actions and queries
Animal#
Average-weight
dateofbirth-species
DeleteAll
escape#
neutered#
species_weight
update Weight
UpdateSpecies
makedecimal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,4 +120,4 @@ I would like to thank everyone!
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 ## 📝 License
 
-This project is [MIT](./LICENSE.md) licensed.
\ No newline at end of file
+This project is [MIT](./LICENSE.md) licensed.
```
Comment (emmanuellekamwa) [inline README.md:125]: - [ ] Your MIT license link is not working. Please kindly fix that.
- [ ] Also make sure that `License`, `Acknowledgement`, and `Show your support` topics have the same appearance as the other topics.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔**
Hello @saluumaa 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (saluumaa) [reply README.md:125]: Fixed with thanks

Comment (rloterh) [reply README.md:125]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Salma,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job implementing recommended changes!

Cheers..Happy coding!💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**